### PR TITLE
`Line::from` for `(Point, Point)`, `(Point, Vec2)`

### PR DESCRIPTION
**Pull request by waywardmonkeys**

### CODE REVIEW

**Review by Philipp-M — APPROVED** [review]
LGTM

File: src/line.rs [hunk 1]
```diff
@@ -90,6 +90,18 @@ impl Line {
     }
 }
 
+impl From<(Point, Point)> for Line {
+    fn from(points: (Point, Point)) -> Self {
```
Comment (DJMcNab) [inline src/line.rs:94]:
```suggestion
    fn from((a, b): (Point, Point)) -> Self {
```

File: src/line.rs [hunk 2]
```diff
@@ -90,6 +90,18 @@ impl Line {
...
+        Line::new(points.0, points.1)
+    }
+}
+
+impl From<(Point, Vec2)> for Line {
+    fn from(pv: (Point, Vec2)) -> Self {
```
Comment (DJMcNab) [inline src/line.rs:100]:
```suggestion
    fn from((origin, direction): (Point, Vec2)) -> Self {
```

**Review by DJMcNab — COMMENTED** [review]
I'd suggest using destructuring here

File: src/line.rs (hunk 1 above)
Comment (waywardmonkeys) [reply src/line.rs:94]: Done, using `(from, to)`

File: src/line.rs (hunk 2 above)
Comment (waywardmonkeys) [reply src/line.rs:100]: Done, using `(origin, displacement)`

**Review by DJMcNab — APPROVED** [review]
Code looks correct once the push is actually done - if this is definitely an API we want, then I'd be happy to land this.

**Comment by waywardmonkeys** [comment]
Will merge today if there's no objections registered.

**Pull request merged**